combat-trainer.lic

```diff
@@ -2507,7 +2508,7 @@ class TrainerProcess
       waitrt?
       fix_standing
     when /^Tactics$/i
-      bput($tactics_actions.sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty?
+      bput($tactics_actions.sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') if game_state.can_engage?
```

Comment:
Note: as written this now prevents this action when `retreating?` which would be a new change. I'm not 100% sure what the `retreating?` flag is about, so maybe this is incorrect. But if we're retreating we want to stay at ranged, and we can't do tactics actions at ranged, right?